pkg/detector/detector.go

```diff
@@ -477,6 +469,17 @@ func (d *ResourceDetector) ApplyPolicy(object *unstructured.Unstructured, object
 		klog.Errorf("Failed to build resourceBinding for object: %s. error: %v", objectKey, err)
 		return err
 	}
+
+	// If this Reconcile action is triggered by Karmada itself and the current bound Policy is lazy activation preference:
+	// * binding exist: resource will delay to sync the placement from Policy to Binding until resource is updated by User.
+	// * binding not exist: a suspended Binding will be created to prevent resource from dispatching until resource is updated by User.
+	if resourceChangeByKarmada && util.IsLazyActivationEnabled(policy.Spec.ActivationPreference) {
+		operationResult = controllerutil.OperationResultNone
+
+		binding.Spec.Suspension = &policyv1alpha1.Suspension{Dispatching: ptr.To[bool](true)}
+		return d.handleLazyActivationBinding(object, binding)
+	}
```

Comment:
This is the tricky part:
1. It's unnecessary to create a ResourceBinding with a suspended state, in addition, it is a waste for the scheduler to handle this ResourceBinding.
2. It feels weird that here just create but not update.